channeld/channeld.c

```diff
@@ -2720,6 +2751,7 @@ static void init_channel(struct peer *peer)
 	u8 *funding_signed;
 	const u8 *msg;
 	u32 feerate_per_kw[NUM_SIDES];
+	u32 minimum_depth;
```

Comment:
Now I may be wrong here, but isn't this the only place where `minimum_depth` field needs be added to channel.
And not to the other ones, i.e. `new_full_channel` and `new_initial_channel`? The `minimum_depth` is only relevant to a committed channel.